src/create-extension-pack.js

```diff
@@ -0,0 +1,184 @@
+/********************************************************************************
+ * Copyright (C) 2021 Ericsson and others.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License v. 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0.
+ *
+ * This Source Code may also be made available under the following Secondary
+ * Licenses when the conditions for such availability set forth in the Eclipse
+ * Public License v. 2.0 are satisfied: GNU General Public License, version 2
+ * with the GNU Classpath Exception which is available at
+ * https://www.gnu.org/software/classpath/license.html.
+ *
+ * SPDX-License-Identifier: EPL-2.0 OR GPL-2.0 WITH Classpath-exception-2.0
+ ********************************************************************************/
+
+/*
+ * Creates an extension package referencing all built-in extensions previously
+ * created during build time i.e. by executing yarn
+ * 
+ * Extensions will be skipped if a corresponding .vsix file is not found under the 'dist'
+ * folder and also not found under the extension registry.
+ */
+// @ts-check
+const fs = require('fs-extra');
+const path = require('path');
+const vsce = require('vsce');
+const yargs = require('yargs');
+
+const { computeVersion, vscodeVersion, isPublished } = require('./version');
+const { dist, extensions, theiaExtension } = require('./paths.js');
+
+const { tag, force } = yargs.option('tag', {
+    choices: ['latest', 'next']
```

Comment:
Can we still create an extension pack at a specific version?

Reply:
The whole creation of vsix packages as well this new creation of extension packs is based on the checked out version of the vscode submodule, then all extensions are bundled in a separate directory ready to be consumed for vsix packaging and now extension packages.
  So there are only two possible options with this setup 'next' version to point to the current commit, used for non released versions, and 'latest' to treat the current commit as an actual release version, so the only difference is the format of the version applied which distinguish actual releases from new commits of unreleased code.

But if we want to build packages for a specific commit, we need to check that one out in the submodule, yarn build it and then execute either the package-vsix.js or create-extension-pack.js.

Reply:
@alvsan09: Very good explanation about the use and consequences of `latest'/'next` CLI parameter. 

If we want to create and __publish__ an extension pack for a specific version, there's a specific way to do it. The main reason is that we do not publish directly to `open-vsx.org` but rather have CI so it - it has a publish token that we do not, permitting the `open-vsx` functional user to be the user listed as having done the publish (distinct from `publisher`). 

The way forward would be to checkout the wanted `vscode submodule` commit/tag and then commit that change in this repo. Then you would force-push that commit to magic branch `ovsx-publish` or `ovsx-publish-next`. 

Here are the workflows that watch/trigger-on these magic branches. IIRC the main if not only difference it the CLI argument with which to call the packaging script:
https://github.com/eclipse-theia/vscode-builtin-extensions/blob/master/.github/workflows/publish-vsx-specific-latest.yml
https://github.com/eclipse-theia/vscode-builtin-extensions/blob/master/.github/workflows/publish-vsx-specific-next.yml